### ISSUE TITLE
Notice bar for posting information to users

### ISSUE THREAD

**Issue by copperwater** [YACS-RCOS/yacs#232]
It would be really nice to have a notice bar so we can inform users of planned downtime, new updates, that they should clear their cache, or anything else. 

This issue entails creating a bar that will sit below the main navbar, probably at full page width, to display this information. Also, a new API resource `/news` should be created that will return a small object that probably will contain a simple string set by the server administrators.

**Comment by Bad-Science** [comment]
For this issue, let's assume the `api/v5/status` resources returns the following data:

```
{
  "message": "hello world",
  "time": "2018-02-21T18:21:02.749Z"
}
```

**Comment by Bad-Science** [comment]
The status bar should also display the time at which the status was created (given by the API)

**Comment by Bad-Science** [comment]
@milnej As of #43 bootstrap 4 is installed. All of the CSS components are available, and the javascript components are available using [ng-bootstrap](https://ng-bootstrap.github.io/#/components/alert/examples)

**Comment by milnej** [comment]
Currently this feature will include notice-bar component, notice service, and notice model. To allow this feature to scale, we are implementing NoticeService with an array of notices called noticeQueue. Every time a notice is dismissed the next one in the queue will be displayed. A dismissed notice will be added to a set of dismissed notices. This set contains unique notice ids so that when the notice-bar loads only non-dismissed notices will be added to the noticequeue to then display.

**Comment by milnej** [comment]
Currently I am getting an interesting error:
ERROR Error: StaticInjectorError(AppModule)[NoticeService -> Array]: 
  StaticInjectorError(Platform: core)[NoticeService -> Array]: 
    NullInjectorError: No provider for Array!
I do not know what this error entails. All of my google attempts have come up with importing HttpClientModule from @angular/common/http. This has not worked though I don't really understand why it would work.

**Comment by milnej** [comment]
The error was caused by incorrectly constructing the noticeQueue. In the notice service I was putting a member variable in the constructor parameters which causes this weird error. When construction an object, it never needs to take in its own member variables.

**Comment by milnej** [comment]
We have decided to move to bootstrap which has allowed the use of bootstrap alerts. The main display of notice bar will be these bootstrap alerts. The issue now is that when we dismiss a given notice we want the next one in the queue to appear. Both dismiss and the next notice functionalities do not work.

**Comment by milnej** [comment]
The basic front end implementation of the notice bar is finished. The pull request will explain the exact implementation.

The API still needs to be completed before we can effectively use Notice Bar.
